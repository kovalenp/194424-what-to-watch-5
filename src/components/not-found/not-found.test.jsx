import React from "react";
import renderer from "react-test-renderer";
import NotFound from "./not-found";

describe(`NotFound meets B21`, () => {

  it(`and rendered correctly for auth'ed`, () => {
    const tree = renderer.create((
      <NotFound />
    )).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
