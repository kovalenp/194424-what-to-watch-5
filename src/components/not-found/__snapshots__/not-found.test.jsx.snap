// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound meets B21 and rendered correctly for auth'ed 1`] = `
<div>
  <h1>
    404 Page Not Found
  </h1>
  <button
    onClick={[Function]}
  >
    Back
  </button>
</div>
`;
